src/ledger.ts

```diff
@@ -76,6 +78,34 @@ export class LedgerCanister {
     );
   };
 
+  public createSubaccount = async ({ name }: { name: string }) => {
```

Comment:
- the return type is missing

- can we use something else than `name`? (see [reserved words](https://www.w3schools.com/js/js_reserved.asp) -> "You should also avoid using the name of JavaScript built-in objects, properties, and methods: ")

Reply:
ok!